bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -241,6 +241,7 @@ void PreProcessor::sendRejectPreProcessReplyMsg(NodeIdType clientId,
 
 template <>
 void PreProcessor::onMessage<ClientPreProcessRequestMsg>(ClientPreProcessRequestMsg *msg) {
+  concord::diagnostics::TimeRecorder scoped_timer(*histograms_.onMessage);
```

Comment:
'onMessage' is a template function, do we combine results for all kind of received messages?

Reply:
If I understood you correctly, it is only for ClientPreProcessRequestMsg.